src/reference/modules/installation/pages/upgrade/notes.adoc

```diff
@@ -26,92 +26,63 @@ rudder package list --all
-* `rudder package check-connection` -> `rudder package update --check`
+Starting 8.2, the generated policies used by the Windows nodes are no longer interpreted by Powershell
+and the variables expansion is now made possible by a more fine grained parsing of the directives and
+techniques content at generation time and a simple template engine.
 
-You can still use the previous implementation with `RUDDER_PKG_COMPAT=1 rudder package ...`, but it will
-be removed in an upcoming release.
+*Using variables on Windows nodes in Rudder will now be done using the exact same syntax as when using some
+on a Linux node. Previously used Windows variables syntaxes are no longer supported and will break generation.*
 
-=== PostgreSQL>=13 version on RHEL 8 (and related distributions)
+As the older syntaxes are no longer supported, after upgrading to 8.2, if you were using the Windows plugin, it
+is most likely that the policy generation will be stopped on your Rudder until you are done with the complete
+syntax migration. A safety mecanism has been added to prevent any generation as long as no longer supported
+syntaxes are in used in your policies.
 
-include::{partialsdir}/rhel_postgres.adoc[]
+As a rule of thumb, avoid any occurrence of the pattern `$(.*)` in your directives and techniques.
+To help with the migration, please refer to the table below:
 
-=== Rudder run now update the policies by default
+[cols="1,1,1"]
+|===
+||Old syntax | New syntax
 
-Until Rudder 7, running `rudder agent run` only ran local policies, and to update the policies
-from the server before running them, users had to add the `-u` flag.
 
-In Rudder 8 the default behavior is now to update the policies before running them.
-A new `-l` (for "local") flag disables the update, and the `-u` flag is now ignored.
+|String type    |$($const.n)                              |${const.n}
+|Json/Data type |$($node.properties.datacenter)           |${node.properties[datacenter]}
+|Json/Data type |$($my_prefix.my_variable.datacenter.dns) |${my_prefix.my_variable[datacenter][dns]}
+|Raw            |$(user_content)                          |${const.dollar}(user_content)
+|===
 
-=== Drop radius support for user authentication
+The generation safety is done by a very simple script located in a new `policy-generation-pre-start` webapp hook
+located in `/opt/rudder/etc/hooks.d/policy-generation-pre-start` and distributed with the Windows plugin.
 
-The support of user authentication through Radius, which was deprecated since Rudder 7, has been removed.
+Note that in some cases the hook may block the generation if you are using inputs similar to the now deprecated
+syntax. If it is the case, please try to use `${const.dollar}(...)` instead of `$(...)` in the directives/methods
+fields.
 
-=== API tokens
+The hook can still be deactivated by commenting its content if needed, with the risk of deploying potentially
```

Comment:
you can just `set -x` it, too

Reply:
How is that gonna disable the hook ?

Reply:
only executable hooks are executed by rudder

Reply:
sorry, I wasn't not clear: I meant to say "you can just remove the executable flag on the hook with chmod -x", and mixed things up